packages/sns/src/utils/ledger.utils.ts

```diff
@@ -0,0 +1,103 @@
+import { Principal } from "@dfinity/principal";
+
+interface SnsAccount {
```

Comment:
maybe, now that we have it in sns-js, it is worth to `export` this interface as well

Reply:
oops, this can be used from the types created from candid. my bad!

Reply:
Damn, not directly because `subaccont: [] | [Subaccount]`.

I created the type in `types/ledger.response.ts` which is exported.